crates/nu-command/src/commands/kill.rs

```diff
@@ -29,6 +30,12 @@ impl WholeStreamCommand for Kill {
                 "process id of process that is to be killed",
             )
             .rest(SyntaxShape::Int, "rest of processes to kill")
+            .named(
+                "signal",
+                SyntaxShape::Int,
```

Comment:
Perhaps should change Int to String (and validate on runtime) so that we can accept signal name as well as signal number, like Bash's `kill`.

Reply:
My personal opinion is that kill(1) mixing argument type is the sort of things Nushell should try to fix (just like we won't match the `-15` dynamic argument style).

If we want a String argument, shouldn't we add another argument (e.g. `--signal-name`)?

Your call, it's not a very strong opinion of mine.

Reply:
yeah, I think separating the name for the number is probably good. Easier to fill it in with a variable and whatnot.

--signalName or if they're all posix, --posixSignal